src/app/components/Home/Home.jsx

```diff
@@ -14,13 +14,11 @@ const Home = ({ sortBy }) => (
     </div>
 
     <div className="content-wrapper">
-      <div className="half">
+      <div className="nypl-column-three-quarters nypl-column-offset-one">
         <p className="lead">Search the New York Public Library Research Catalog
           for materials available to use in one of four research libraries located
```

Comment:
per @courtneymcgee and the toolkit the text should be “Search The New York Public Library's world-renowned collections for items available for use at our research centers. Be sure to request materials in advance to make the most of your time on site.”

“research centers” should link to https://www.nypl.org/locations/map?libraries=research

“request materials” should link to https://www.nypl.org/help/request-research-materials